libraries/ms-helper/inc/thermistor.h

```diff
@@ -28,8 +28,8 @@ StatusCode thermistor_init(ThermistorStorage *storage, GPIOAddress thermistor_gp
                            ThermistorPosition position);
 
 // Fetch the temperature reading in milliCelsius from the MCU's ADC
-StatusCode thermistor_get_temp(ThermistorStorage *storage, uint32_t *temperature_millicelcius);
+StatusCode thermistor_get_temp(ThermistorStorage *storage, uint16_t *temperature_millicelcius);
 
-// Calculate the temperature in milliCelsius from milliohms
-StatusCode thermistor_calculate_temp(uint32_t thermistor_resistance_milliohms,
-                                     uint32_t *temperature_millicelcius);
+// Calculate the temperature in hundreds of milliCelsius from ohms
```

Comment:
That's confusing. You should just use 0.1C or something like that. Your variable name is out of date as well.